packages/design-system/src/types/action-dropdown.ts

```diff
@@ -0,0 +1,11 @@
+import type { KeyboardShortcut } from '@/types/keyboardshortcut';
+
+export interface IActionDropdownItem {
```

Comment:
Nit: I think we stopped using `I` prefix for types and interfaces, think we should work on getting rid of them.